packages/devtools_app/web/debugger_screen.html

```diff
@@ -0,0 +1,152 @@
+
+    <script src="packages/ansi_up/ansi_up.js"></script>
+
+    <link href="packages/codemirror/codemirror.css" rel="stylesheet">
+    <script src="packages/codemirror/codemirror.js"></script>
+
+    <script src="packages/devtools_app/src/ui/plotly.js"></script>
+    <script type="text/javascript" src="packages/plotly_js/plotly.min.js"></script>
+
+    <script type="text/javascript" src="packages/split/split.min.js"></script>
+</head>
+
+<body layout vertical style="width: 100%; height: 100%; background-color: transparent;">
+
+    <header flex wrap none class="masthead legacy-dart" id="header" layout horizontal hidden>
+        <a class="masthead-logo" id="title">
+            <span class="optional-550">Dart DevTools</span>
+        </a>
+
+        <div flex horiz-padding></div>
+
+        <nav class="masthead-nav" id="main-nav">
+            <!-- add the known pages -->
+            <a disabled>
+                <span class="octicon octicon-device-mobile"></span>
+                <span>
+                    <span class="optional-1060">Flutter Inspector</span>
+                </span>
+            </a>
+            <a disabled>
+                <span class="octicon octicon-pulse"></span>
+                <span>
+                    <span class="optional-1060">Timeline</span>
+                </span>
+            </a>
+            <a disabled>
+                <span class="octicon octicon-dashboard"></span>
+                <span>
+                    <span class="optional-1060">Performance</span>
+                </span>
+            </a>
+            <a disabled>
+                <span class="octicon octicon-package"></span>
+                <span>
+                    <span class="optional-1060">Memory</span>
+                </span>
+            </a>
+            <a disabled>
+                <span class="octicon octicon-bug"></span>
+                <span>
+                    <span class="optional-1060">Debugger</span>
+                </span>
+            </a>
+            <a disabled>
+                <span class="octicon octicon-clippy"></span>
+                <span>
+                    <span class="optional-1060">Logging</span>
+                </span>
+            </a>
+        </nav>
+
+        <div id="global-actions" class="masthead-nav"></div>
+
+        <div class="masthead-nav">
+            <span horiz-padding class="masthead-item">•</span>
+            <div class="masthead-item action-button active" id="icon-nav">
+                <span class="octicon octicon-info"></span>
+            </div>
+            <div class="masthead-item action-button active" id="send-feedback-button" title="Send Feedback">
+                <span class="octicon octicon-comment"></span>
+            </div>
+        </div>
+    </header>
+
+    <div id="messages-container" class="container legacy-dart"></div>
+
+    <form id="ga-dialog" class="container legacy-dart"></form>
+    <form id="connect-dialog" class="container legacy-dart"></form>
```

Comment:
remove this as well. Basically strip this file way down to just what is needed for the debugger page.

Reply:
See previous comment.